### PR TITLE
Added Microsoft Edge browser support

### PR DESCRIPTION
**Pull request by marcoaam**

### CODE REVIEW

File: src/Coypu.AcceptanceTests/ApiExamples.cs [hunk 1]
```diff
@@ -646,6 +646,7 @@ private static RemoteWebDriver CustomProfile()
 
         [TestCase("Windows 7", "firefox", "25")]
         [TestCase("Windows XP", "internet explorer", "6")]
+        [TestCase("Windows 10", "MicrosoftEdge", "14")]
         public void CustomBrowserSession(string platform, string browserName, string version)
```
Comment (adiel) [inline src/Coypu.AcceptanceTests/ApiExamples.cs]: HI thanks for this! Did you actually run this test? I would have thought you'd get `BrowserNotSupported` unless you also added some code to construct the correct selenium driver here:

https://github.com/featurist/coypu/blob/c86ac64d3f82aeb184eeeca1ee10b6b9850e86f1/src/Coypu/Drivers/Selenium/DriverFactory.cs

File: Coypu.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>Coypu</id>
-    <version>2.11.0</version>
+    <version>2.12.0</version>
```
Comment (adiel) [inline Coypu.nuspec]: Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR

File: History.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+# Version 2.12.0
+
+### Updated
+
+* Changes to support Microsoft Edge browser
+
+Release date: 2016-12-22
+
 # Version 2.11.0
```
Comment (adiel) [inline History.md]: Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR

File: src/Coypu.AcceptanceTests/ApiExamples.cs (hunk 1 above)
Comment (marcoaam) [inline src/Coypu.AcceptanceTests/ApiExamples.cs]: Hi, Yes I did. The way the test is implemented it doesn't require changes to the DriverFactory.cs as in this case it is passing in the RemoteWebDriver in the constructor so it is not making use of the DriverFactory.cs, which is the same case that I have in my codebase. There is no harm in adding it though. I can submit another pull request with the changes and remove the commits of the bump of the version and the history.md changes if you prefer.

**Comment by marcoaam** [comment]
I have added support for MicrosoftEdge now in the DriverFactory.cs as well and added a couple of tests to cover it. You will need IE driver and MicrosoftWebDriver in your bin folder to be able to run these new tests.

**Comment by Jetski5822** [comment]
Thanks

**Pull request merged**